client/analytics/lib/analyticsappview.js

```diff
@@ -13,6 +13,12 @@ export default class AnalyticsAppView extends JView {
 
   pistachio () {
     // return 'sinan\'s analytics'
-    return '<iframe src="http://dev.koding.com:8090/countly" frameborder="0" width="100%" height="100%" border="0"></iframe>'
+    
+    // TODO: replace username with group.slug and replace apiKey with group's api key
+    var username =  'admin'
```

Comment:
@sinan we need to get the real info with `remote.api.JGroupData.fetchByKey 'countly', callback`